# Week 3.3: Multi-phase chemistry

## Gas Mixtures and Partial Pressures

In explaining ideal gas law, we focus entirely on the properties of pure gases with only a single chemical species. But what happens when two or more gases are mixed? 
In this section, we describe how to determine the contribution of each gas present to the total pressure of the mixture.

### Partial Pressures:
The ideal gas law assumes that all gases behave identically and that their behavior is independent of attractive and repulsive forces. If volume and temperature are held constant, the ideal gas equation can be rearranged to show that the pressure of a sample of gas is directly proportional to the number of moles of gas present:

**_P = n(RT/V) = n × const._**

Nothing in the equation depends on the nature of the gas — only the amount.

With this assumption, let’s suppose we have a mixture of two ideal gases that are present in equal amounts. What is the total pressure of the mixture?

Because the pressure depends on only the total number of particles of gas present, the total pressure of the mixture will simply be twice the pressure of either component. More generally, the total pressure exerted by a mixture of gases at a given temperature and volume is the sum of the pressures exerted by each gas alone. Furthermore, if we know the volume, the temperature, and the number of moles of each gas in a mixture, then we can calculate the pressure exerted by each gas individually, which is its partial pressure, the pressure the gas would exert if it were the only one present (at the same temperature and volume).

To summarize, the total pressure exerted by a mixture of gases is the sum of the partial pressures of component gases. This law was first discovered by John Dalton, the father of the atomic theory of matter. It is now known as Dalton’s law of partial pressures. We can write it mathematically as

**_P<sub>tot</sub> = P<sub>1</sub> + P<sub>2</sub> + P<sub>3</sub> + P<sub>4</sub> …_**

**_P<sub>tot</sub> = ∑<sub>i=1</sub><sup>n</sup> P<sub>i</sub>_**

where P<sub>tot</sub> is the total pressure and the other terms are the partial pressures of the individual gases (up to n component gases).

**Example:**
Deep-sea divers must use special gas mixtures in their tanks, rather than compressed air, to avoid serious problems, most notably a condition called “the bends.” At depths of about 350 ft, divers are subject to a pressure of approximately 10 atm. A typical gas cylinder used for such depths contains 51.2 g of O<sub>2</sub> and 326.4 g of He and has a volume of 10.0 L. What is the partial pressure of each gas at 20°C, and what is the total pressure in the cylinder at this temperature?


In [11]:
# Deep-sea diver cylinder: partial pressures and total pressure (ideal gas law)

# Given
m_He_g = 326.4
m_O2_g = 51.2
M_He = 4.003      # g/mol
M_O2 = 32.00      # g/mol

V_L = 10.0
T_C = 20.0
T_K = 273.15 + T_C

R = 0.08206  # (atm·L)/(mol·K)

# A) moles
n_He = m_He_g / M_He
n_O2 = m_O2_g / M_O2

# B) partial pressures
P_He = (n_He * R * T_K) / V_L
P_O2 = (n_O2 * R * T_K) / V_L

# total pressure
P_tot = P_He + P_O2

print(f"n_He = {n_He:.2f} mol")
print(f"n_O2 = {n_O2:.2f} mol")
print(f"P_He = {P_He:.1f} atm")
print(f"P_O2 = {P_O2:.2f} atm")
print(f"P_tot = {P_tot:.2f} atm")


n_He = 81.54 mol
n_O2 = 1.60 mol
P_He = 196.1 atm
P_O2 = 3.85 atm
P_tot = 200.00 atm


**Solution:**

A The number of moles of He is

n<sub>He</sub> = 326.4 g / (4.003 g/mol) = 81.54 mol

The number of moles of O<sub>2</sub> is

n<sub>(O<sub>2</sub>)</sub> = 51.2 g / (32.00 g/mol) = 1.60 mol

B We can now use the ideal gas law to calculate the partial pressure of each:

P<sub>He</sub> = (n<sub>He</sub> RT)/V = (81.54 mol × 0.08206 (atm·L)/(mol·K) × 293.15 K) / 10.0 L = 196.2 atm

P<sub>(O<sub>2</sub>)</sub> = (n<sub>(O<sub>2</sub>)</sub> RT)/V = (1.60 mol × 0.08206 (atm·L)/(mol·K) × 293.15 K) / 10.0 L = 3.85 atm

The total pressure is the sum of the two partial pressures:

P<sub>tot</sub> = P<sub>He</sub> + P<sub>(O<sub>2</sub>)</sub> = (196.2 + 3.85) atm = 200.00 atm


**Problem:**
A cylinder of compressed natural gas has a volume of 20.0 L and contains 1813 g of methane and 336 g of ethane. Calculate the partial pressure of each gas at 22.0°C and the total pressure in the cylinder.


## Mole Fractions of Gas Mixtures:

The composition of a gas mixture can be described by the mole fractions of the gases present.

The mole fraction (χ) of any component of a mixture is the ratio of the number of moles of that component to the total number of moles of all the species present in the mixture (n<sub>tot</sub>):

χ<sub>A</sub> = "moles of A" / "total moles" = n<sub>A</sub>/n<sub>tot</sub> = n<sub>A</sub>/(n<sub>A</sub> + n<sub>B</sub> + ⋯)

The mole fraction is a dimensionless quantity between 0 and 1. If χ<sub>A</sub> = 1.0, then the sample is pure A, not a mixture. If χ<sub>A</sub> = 0, then no A is present in the mixture. The sum of the mole fractions of all the components present must equal 1.

To see how mole fractions can help us understand the properties of gas mixtures, let’s evaluate the ratio of the pressure of a gas A to the total pressure of a gas mixture that contains A. We can use the ideal gas law to describe the pressures of both gas A and the mixture:

P<sub>A</sub> = (n<sub>A</sub>RT)/V  and  P<sub>tot</sub> = (n<sub>tot</sub>RT)/V.

The ratio of the two is thus

P<sub>A</sub>/P<sub>tot</sub> = ((n<sub>A</sub>RT)/V)/((n<sub>tot</sub>RT)/V) = n<sub>A</sub>/n<sub>tot</sub> = χ<sub>A</sub>

Rearranging this equation gives

P<sub>A</sub> = χ<sub>A</sub> P<sub>tot</sub>

That is, the partial pressure of any gas in a mixture is the total pressure multiplied by the mole fraction of that gas.

**Dalton’s Law, or the Law of Partial Pressures,** states that the total pressure exerted by a mixture of gases is equal to the sum of the partial pressures of the gases in the mixture.

P<sub>total</sub> = P<sub>A</sub> + P<sub>B</sub> + ...


## Activity Coefficients:

In descriptions of the properties of the components of liquid mixtures, rational activity coefficients are given the symbols f<sub>1</sub>, f<sub>2</sub>, f<sub>3</sub>…..

With reference to solvents their thermodynamic properties are described by rational activity coefficients f<sub>1</sub>, f<sub>2</sub>, f<sub>3</sub>… and (practical) osmotic coefficient, ϕ.

The properties of solutes in solutions are described using activity coefficients which are linked to the descriptions of the composition of solutions: molality scale, γ<sub>j</sub>; concentration scale; mole fraction scale f<sub>j</sub><sup>*</sup>.

These coefficients are intimately related to the concept of activity. Their significance is clarified by equations relating chemical potentials to the composition of a given system;

- components in liquid mixtures,
- solvents in solutions and
- solutes in solutions.

In all cases they summarize the extent to which the thermodynamic properties of liquid mixtures and solutions are not ideal.

The challenge for chemists is to understand in terms of molecular properties why the thermodynamic properties of real systems are not ideal.


## Fugacity and Fugacity Coefficient:

For an ideal gas, the Gibbs free energy is a simple function of its pressure. It turns out to be useful to view the integral as a contribution to a “corrected pressure.” The “correction” is an adjustment to the pressure that, in our calculations, makes the real gas behave as an ideal gas.

The idea is that we can express the Gibbs free energy as a function of this corrected pressure, which is termed as fugacity f.

Fugacity is therefore a function of pressure. Fugacity also has the units of pressure, which we always take to be bars. We define the fugacity of A at pressure P, f<sub>A</sub>(P), by

Ḡ<sub>A</sub>(P) = Δ<sub>f</sub>G<sub>A</sub><sup>o</sup>(HIG<sup>o</sup>) + RT ln( f<sub>A</sub>(P) / f<sub>A</sub>(HIG<sup>o</sup>) )

The fugacity, f<sub>A</sub>(HI<sup>G</sup><sup>o</sup>), and the standard Gibbs free energy of formation, Δ<sub>f</sub>G<sub>A</sub><sup>o</sup>(HIG<sup>o</sup>), describe the same state. We define the fugacity in this standard state to be one bar. That is, the fugacity of a substance in its hypothetical ideal-gas standard state, f<sub>A</sub>(HIG<sup>o</sup>), is a constant whose value is 1 bar.

(If we want to express fugacity in units of, say, pascals, then P<sup>o</sup> = f<sub>A</sub>(HI<sup>G</sup><sup>o</sup>) = 10<sup>5</sup> Pa.)

We can calculate the fugacity of a real gas, at pressure P and temperature T, from

RT ln( f<sub>A</sub>(P) / f<sub>A</sub>(HI<sup>G</sup><sup>o</sup>) ) = RT ln( P / P<sub>0</sub> ) + RT∫<sub>0</sub><sup>P</sup>[ V̄/RT − 1/P ] dP

We find it useful to introduce a function of pressure that we call the fugacity coefficient, γ<sub>A</sub>(P). We define it as

γ<sub>A</sub>(P) = f<sub>A</sub>(P) / P


## The Law of Chemical Equilibrium:


If we write an equation for a gaseous equilibrium in general in the form:

a A(g) + b B(g) ⇌ c C(g) + d D(g)

then the equilibrium constant defined by the equation,

K<sub>c</sub> = ([C]<sup>c</sup> [D]<sup>d</sup>) / ([A]<sup>a</sup> [B]<sup>b</sup>)

K<sub>c</sub> is a constant quantity depending only on the temperature and the nature of the reaction. This general result is called the law of chemical equilibrium, or the law of mass action.


**Example:** Write expressions for the equilibrium constant for the following reactions:

2HI(g) ⇌ H<sub>2</sub>(g) + I<sub>2</sub>(g)

N<sub>2</sub>(g) + 3H<sub>2</sub>(g) ⇌ 2NH<sub>3</sub>(g)

O<sub>2</sub>(g) + 4HCl(g) ⇌ 2H<sub>2</sub>O(g) + 2Cl<sub>2</sub>(g)





Solution:

K<sub>c</sub> = ([H<sub>2</sub>] [I<sub>2</sub>]) / [HI]<sup>2</sup>

K<sub>c</sub> = [NH<sub>3</sub>]<sup>2</sup> / ([N<sub>2</sub>] [H<sub>2</sub>]<sup>3</sup>)

K<sub>c</sub> = ([H<sub>2</sub>O]<sup>2</sup> [Cl<sub>2</sub>]<sup>2</sup>) / ([O<sub>2</sub>] [HCl]<sup>4</sup>)


**Example:** A mixture containing equal concentrations of methane and steam is passed over a nickel catalyst at 1000 K. The emerging gas has the composition [CO] = 0.1027 mol dm–3, [H2] = 0.3080 mol dm<sup> –3</sup>, and [CH4] = [H2O] = 0.8973 mol dm <sup>–3</sup>. Assuming this mixture is at equilibrium, calculate the equilibrium constant K<sub>c </sub>for the reaction.

In [4]:
# Compute Kc for: CH4 + H2O ⇌ CO + 3H2



Kc = 3.726896e-03
Kc = 3.7269 × 10^-3


**Note** The yield of H<sub>2</sub> at this temperature is quite poor. In the commercial production of H<sub>2</sub> from natural gas, the reaction is run at a somewhat higher temperature where the value K<sub>c</sub> is larger.


## Henry’s Law:


Henry's law is one of the gas laws formulated by **William Henry (1803)**.


It states that:

> “<b>At a constant temperature, the amount of a given gas that dissolves in a given type and volume of liquid is directly proportional to the partial pressure of that gas in equilibrium with that liquid.</b>”


An equivalent way of stating the law is that the solubility of a gas in a liquid is directly proportional to the partial pressure of the gas above the liquid: <br> C=kP<sub>gas</sub>

where
- **C** is the solubility of a gas at a fixed temperature in a particular solvent (in units of **M** or **mL gas/L**)
- **k** is Henry's law constant (often in units of **M/atm**)
- **P<sub>gas</sub>** is the partial pressure of the gas (often in units of **Atm**)



![Figure 1](figures/figure_1.png)
<br><b>Figure 1: </b>Positive non-ideal behavior of the vapor pressure of a solution follows Henry's Law at low concentrations and Raoult’s law at high concentrations (pure).


<b> Example: 1</b> What is Henry's constant for neon dissolved in water given: **_C<sub>Ne</sub>_**=**23.5 mL⁄L** solution and STP (**22,414 mL/mole gas**) and pressure (**1 atm**)?


In [1]:
# Example 1 



C = 0.0010484518604443652 M
k = 0.0010484518604443652 M/atm


<b> Solution </b> Now we can rearrange our equation from above to solve for the constant:

**_C = kP<sub>Ne</sub>_**

To use **_C_**, we must convert **23.5 mL/L** solution to Molarity. Since **Ne** is a gas, we can use standard molar volume. Thus, giving us:

(**23.5 mL/L soln**) (**1 mole Ne/22,414 mL**) = **0.00105 M**.

Now we have solved for the solubility of **Ne** in the solution. **_C_** = **0.00105 M** and we know the pressure at STP is **1 atm**, so we can now use our rearranged equation:

**_k = C / P<sub>Ne</sub>_** 

Where **_C_** = **0.00105 M**, **_P<sub>Ne</sub>_** = **1 atm**, thus giving us **k = 0.00105 M/atm**.


<b>Example: 2</b> Compute the molar solubility in water that is saturated with air.


In [2]:
# Example 2 (coded)



C = 3.5804999999999993e-05 M
OR
C= 3.580e-05 ^M


<b> Solution </b>

Use constant (**k**) that we just calculated and **P<sub>Ne</sub>** in air.

**_C = kP<sub>gas</sub>_** 

C= (**0.00105** M/atm) (**0.0341** atm)

C= 3.58* 10<sup>-5</sup> M


<b> Applicability of Henry’s law: </b>
- Henry's law only works if the molecules are at equilibrium.
- Henry's law does not work for gases at high pressures (e.g., N<sub>2(g)</sub>  at high pressure becomes very soluble and harmful when in the blood supply).
- Henry's law does not work if there is a chemical reaction between the solute and solvent (e.g., HCl <sub>(g)</sub> reacts with water by a dissociation reaction to generate H<sub>3</sub>O<sup>+</sup> and Cl<sup>-</sup> ions).


## Raoult’s Law:


<b> Raoult's law </b> states that the vapor pressure of a solvent above a solution is equal to the vapor pressure of the pure solvent at the same temperature scaled by the mole fraction of the solvent present: </br>

<b><i> P<sub>solution</sub> = χ<sub>solvent</sub> P<sup> o</sup><sub>solvent</sub> </b></i> </br> </br>
In the 1880s, French chemist François-Marie Raoult discovered that when a substance is dissolved in a solution, the vapor pressure of the solution will generally decrease. This observation depends on two variables:
  1. the mole fraction of the amount of dissolved solute present and
  2. the original vapor pressure (pure solvent).
 </br>At any given temperature for a particular solid or liquid, there is a pressure at which the vapor formed above the substance is in dynamic equilibrium with its liquid or solid form. This is the vapor pressure of the substance at that temperature. At equilibrium, the rate at which the solid or liquid evaporates is equal to the rate that the gas is condensing back to its original form. All solids and liquids have a vapor pressure, and this pressure is constant regardless of how much of the substance is present. </br>
Raoult's Law only works for ideal solutions.
>"An ideal solution shows thermodynamic mixing characteristics identical to those of ideal gas mixtures [except] ideal solutions have intermolecular interactions equal to those of the pure components."
>
Like many other concepts explored in Chemistry, Raoult's Law only applies under ideal conditions in an ideal solution. However, it still works fairly well for the solvent in dilute solutions. In reality though, the decrease in vapor pressure will be greater than that calculated by Raoult's Law for extremely dilute solutions. ![Figure 1](figures/figure_1.png) </br>Figure: Positive non-ideal behavior of the vapor pressure of a solution follows Henry's Law at low concentrations and Raoult's Law at high concentrations (pure).


<b> Limitations on Raoult's Law </b>
In practice, there's no such thing as an ideal solution! However, features of one include:
 - Ideal solutions satisfy Raoult's Law. 
 - In an ideal solution, it takes exactly the same amount of energy for a solvent molecule to break away from the surface of the solution as it did in the pure solvent. The forces of attraction between solvent and solute are exactly the same as between the original solvent molecules - not a very likely event!


<b> Example: 1 </b> 
1.5 moles of cherry Kool-Aid are added to a pitcher containing 2 liters of water on a nice day at 25<sup>o</sup>C. The vapor pressure of water alone is 23.8 mm Hg at 25<sup>o </sup>C. What is the new vapor pressure of Kool-Aid?


In [3]:
# Raoult’s Law example: Kool-Aid in water (25oC)

# Given



P_H2O = 23.8 mm Hg
Mass of water = 2000 g
Moles H2O = 111.0 moles
χ_H2O = 0.987
P_(Kool-Aid) = χ_H2O * P_H2O = 23.5 mm Hg


<b> Solution </b>

P<sub>H<sub>2</sub>O</sub> = 23.8 mm Hg

To solve for the mole fraction, you must first convert the 2 L of water into moles:

1 L = 1000 mL = 1000 g;

Knowing this, we can convert the mass of water (2000 g) into moles: 2000 g / 18.02 g (molar mass of water) = 110.9 moles H<sub>2</sub>O

Solve for the mole fraction, χ<sub>H<sub>2</sub>O</sub> :

χ<sub>H<sub>2</sub>O</sub> = moles H<sub>2</sub>O / total moles = 110.9 moles / 110.9 + 1.5 moles = 0.979

Finally, apply Raoult's Law

P<sub>(Kool-Aid)</sub>=χ<sub>H<sub>2</sub>O</sub> P<sub>H<sub>2</sub>O</sub> = (0.979) (23.8 mm Hg) = 23.5 mm Hg


<b> Example 2: </b>
Calculate the vapor pressure of a solution made by dissolving 50.0 g glucose, C<sub>6</sub> H<sub>12</sub>O<sub>6</sub>, in 500 g of water. The vapor pressure of pure water is 47.1 torr at 37°C.


In [4]:
# Raoult’s Law example 2: Glucose in water (37°C)

# Given


n_water = 27.8 mol
n_glucose = 0.277 mol
chi_solvent = 0.99
P_solution = 0.99 × 47.1 = 46.63 torr


<b> Solution </b>

To use Raoult's Law Equation, we need to calculate the mole fraction of water (the solvent) in this sugar-water solution.

χ<sub>solvent</sub> = "moles of water" / ("moles of solute" + "moles of solvent")

χ<sub>solvent</sub> = n<sub>water</sub> / (n<sub>glucose</sub> + n<sub>water</sub>)

The molar mass of glucose if 180.2 g/mol and of water is 18 g/mol.
So,

n<sub>water</sub> = 500 g / (18 g/mol) = 27.7 mol

and

n<sub>glucose</sub> = 50 g / (180.2 g/mol) = 0.277 mol

and

χ<sub>solvent</sub> = 27.7 mol / (0.277 mol + 27.7 mol) = 0.99

Note that this still relatively dilute.

P<sub>solution</sub> = 0.99 × 47.1 = 46.63 torr


<b> Example 3: </b>
What are the partial pressures of benzene and toluene in a solution in which the mole fraction of benzene is 0.6? What is the total vapor pressure? The vapor pressure of pure benzene is 95.1 mm Hg and the vapor pressure of pure toluene 28.4 mm Hg at 25<sup> o </sup>C.


In [5]:
# Example 3: Benzene + Toluene partial pressures (Raoult's Law)



chi_benzene = 0.6
chi_toluene = 0.4
P_benzene = (0.6) (95.1 mm Hg) = 57.1 mm Hg
P_toluene = (0.4) (28.4 mm Hg) = 11.4 mm Hg
P_Total = 57.1 mm Hg + 11.4 mm Hg = 68.4 mm Hg


<b> Solution </b>


If  χ<sub>benzene</sub> = 0.6, then  χ<sub>toluene</sub> = 0.4, because  1 − 0.6 = 0.4

We know the mole fractions and vapor pressures:

P<sub>benzene</sub> = χ<sub>benzene</sub> P<sub>benzene</sub> = (0.6) (95.1 mm Hg) = 57.1 mm Hg

P<sub>toluene</sub> = χ<sub>toluene</sub> P<sub>toluene</sub> = (0.4) (28.4 mm Hg) = 11.4 mm Hg

The total vapor pressure is simply the sum of the partial pressures:

P<sub>Total</sub> = P<sub>benzene</sub> + P<sub>toluene</sub> = 57.1 mm Hg + 11.4 mm Hg = 68.4 mm Hg



<b> References </b>

1. Petrucci, et al. *General Chemistry Principles &amp; Modern Applications.* 9th ed. Upper Saddle River, NJ: Pearson Prentice Hall, 2007.  
2. Kugel, Roger W. "Raoult's Law: Binary Liquid-Vapor Phase Diagrams" *Journal of Chemical Education* **75**.9 (1998): 1125.  
3. P. Tzias, C. Treiner, and M. Chemla. “Applicability of Raoult’s Law in Nonideal Mixed Solvents” *Journal of Solution Chemistry* **6**.6 (1977): 393-402.  
4. <https://chem.libretexts.org/Bookshelves/Physical_and_Theoretical_Chemistry_Textbook_Maps/Supplemental_Modules_(Physical_and_Theoretical_Chemistry)/Physical_Properties_of_Matter/Solutions_and_Mixtures/Ideal_Solutions/Changes_In_Vapor_Pressure_Raoult's_Law>


## The Solubility Product:


When a slightly soluble ionic compound is added to water, some of it dissolves to form a solution, establishing an equilibrium between the pure solid and a solution of its ions. For the dissolution of calcium phosphate, one of the two main components of kidney stones, the equilibrium can be written as follows, with the solid salt on the left:

Ca<sub>3</sub>(PO<sub>4</sub>)<sub>2</sub> (s) ⇌ 3Ca<sup>2+</sup> (aq) + 2PO<sub>4</sub><sup>3−</sup> (aq)

The basic anions, such as S<sup>2−</sup>, PO<sub>4</sub><sup>3−</sup>, and CO<sub>3</sub><sup>2−</sup>, react with water to produce OH<sup>−</sup> and the corresponding protonated anion. Consequently, their calculated molarities, assuming no protonation in aqueous solution, are only approximate.

The equilibrium constant for the dissolution of a sparingly soluble salt is the solubility product (K<sub>sp</sub>) of the salt. Because the concentration of a pure solid such as Ca<sub>3</sub>(PO<sub>4</sub>)<sub>2</sub> is a constant, it does not appear explicitly in the equilibrium constant expression.

The equilibrium constant expression for the dissolution of calcium phosphate is therefore:

K = ([Ca<sup>2+</sup>]<sup>3</sup> [PO<sub>4</sub><sup>3−</sup>]<sup>2</sup>) / [Ca<sub>3</sub>(PO<sub>4</sub>)<sub>2</sub>]

[Ca<sub>3</sub>(PO<sub>4</sub>)<sub>2</sub>]K = K<sub>sp</sub> = [Ca<sup>2+</sup>]<sup>3</sup> [PO<sub>4</sub><sup>3−</sup>]<sup>2</sup>

At 25°C and pH 7, K<sub>sp</sub> for calcium phosphate is 2.07 × 10<sup>−33</sup>, indicating that the concentrations of Ca<sup>2+</sup> and PO<sub>4</sub><sup>3−</sup> ions in solution that are in equilibrium with solid calcium phosphate are very low.

Solubility products are determined experimentally by directly measuring either the concentration of one of the component ions or the solubility of the compound in a given amount of water. However, whereas solubility is usually expressed in terms of mass of solute per 100 mL of solvent, K<sub>sp</sub>, like K, is defined in terms of the molar concentrations of the component ions.


**Example** 
Calcium oxalate monohydrate [Ca(O<sub>2</sub>CCO<sub>2</sub>)·H<sub>2</sub>O, also written as CaC<sub>2</sub>O<sub>4</sub>·H<sub>2</sub>O] is a sparingly soluble salt that is the other major component of kidney stones [along with Ca<sub>3</sub>(PO<sub>4</sub>)<sub>2</sub>]. Its solubility in water at 25°C is 7.36 × 10<sup>−4</sup> g/100 mL. Calculate its K<sub>sp</sub>.


In [5]:
# Calcium oxalate monohydrate Ksp from solubility

# Given
solubility_g_per_100mL = 7.36e-4   # g / 100 mL
molar_mass = 146.1                 # g/mol  (CaC2O4·H2O)
volume_100mL_L = 0.100             # L

# Moles dissolved in 100 mL
moles_in_100mL = solubility_g_per_100mL / molar_mass

# Convert to molarity (mol/L) for the saturated solution
M = moles_in_100mL / volume_100mL_L  # [Ca2+] = [ox2-] = M

# Ksp = [Ca2+][ox2-] = M^2
Ksp = M * M

print(f"moles in 100 mL = {moles_in_100mL:.3e} mol")
print(f"[Ca2+] = [ox2-] = {M:.3e} M")
print(f"Ksp = {Ksp:.3e}")

# Print as mantissa × 10^exponent (like notes)
import math
exp10 = math.floor(math.log10(abs(Ksp))) if Ksp != 0 else 0
mant = Ksp / (10**exp10) if Ksp != 0 else 0
print(f"Ksp = {mant:.3f} × 10^{exp10}")


moles in 100 mL = 5.038e-06 mol
[Ca2+] = [ox2-] = 5.038e-05 M
Ksp = 2.538e-09
Ksp = 2.538 × 10^-9



**Solution:** We need to write the solubility product expression in terms of the concentrations of the component ions. For calcium oxalate monohydrate, the balanced dissolution equilibrium and the solubility product expression (abbreviating oxalate as ox<sup>2−</sup>) are as follows:

Ca(O<sub>2</sub>CCO<sub>2</sub>)·H<sub>2</sub>O(s) ⇌ Ca<sup>2+</sup>(aq) + ox<sup>2−</sup>(aq) + H<sub>2</sub>O(l)  K<sub>sp</sub> = [Ca<sup>2+</sup>][ox<sup>2−</sup>]

Neither solid calcium oxalate monohydrate nor water appears in the solubility product expression because their concentrations are essentially constant.

Next, we need to determine [Ca<sup>2+</sup>] and [ox<sup>2−</sup>] at equilibrium. We can use the mass of calcium oxalate monohydrate that dissolves in 100 mL of water to calculate the number of moles that dissolve in 100 mL of water. From this, we can determine the number of moles that dissolve in 1 L of water. For dilute solutions, the density of the solution is nearly the same as that of water, so dissolving the salt in 1 L of water gives essentially 1 L of solution. Because each 1 mol of dissolved calcium oxalate monohydrate dissociates to produce 1 mol of calcium ions and 1 mol of oxalate ions, we can obtain the equilibrium concentrations that must be inserted into the solubility product expression. The number of moles of calcium oxalate monohydrate that dissolve in 100 mL of water is as follows:

(7.36 × 10<sup>−4</sup> g) / (146.1 g/mol) = 5.04 × 10<sup>−6</sup> mol  Ca(O<sub>2</sub>CCO<sub>2</sub>)·H<sub>2</sub>O

The number of moles of calcium oxalate monohydrate that dissolve in 1.00 L of the saturated solution is as follows:

((5.04 × 10<sup>−6</sup> mol Ca(O<sub>2</sub>CCO<sub>2</sub>)·H<sub>2</sub>O) / 100 mL) (1000 mL / 1.00 L) = 5.04 × 10<sup>−5</sup> mol/L = 5.04 × 10<sup>−5</sup> M

Because of the stoichiometry of the reaction, the concentration of Ca<sup>2+</sup> and ox<sup>2−</sup> ions are both 5.04 × 10<sup>−5</sup> M. Inserting these values into the solubility product expression,

K<sub>sp</sub> = [Ca<sup>2+</sup>][ox<sup>2−</sup>] = (5.04 × 10<sup>−5</sup>)(5.04 × 10<sup>−5</sup>) = 2.54 × 10<sup>−9</sup>

In our calculation, we have ignored the reaction of the weakly basic anion with water, which tends to make the actual solubility of many salts greater than the calculated value.



**Problem:** One crystalline form of calcium carbonate (CaCO<sub>3</sub>) is the mineral sold as “calcite” in mineral and gem shops. The solubility of calcite in water is 0.67 mg/100 mL. Calculate its K<sub>sp</sub>.


## The Ion Product:


The ion product (Q) of a salt is the product of the concentrations of the ions in solution raised to the same powers as in the solubility product expression. It is analogous to the reaction quotient (Q) discussed for gaseous equilibria. Whereas K<sub>sp</sub> describes equilibrium concentrations, the ion product describes concentrations that are not necessarily equilibrium concentrations.

The ion product Q is analogous to the reaction quotient Q for gaseous equilibria.

As summarized in Figure below, there are three possible conditions for an aqueous solution of an ionic solid:

- Q &lt; K<sub>sp</sub>. The solution is unsaturated, and more of the ionic solid, if available, will dissolve.
- Q = K<sub>sp</sub>. The solution is saturated and at equilibrium.
- Q &gt; K<sub>sp</sub>. The solution is supersaturated, and ionic solid will precipitate. </br>
![Figure 1](figures/figure_2.png) </br>
**Figure:The Relationship between Q and K<sub>sp</sub>. If Q is less than K<sub>sp</sub>, the solution is unsaturated and more solid will dissolve until the system reaches equilibrium (Q = K<sub>sp</sub>). If Q is greater than K<sub>sp</sub>, the solution is supersaturated and solid will precipitate until Q = K<sub>sp</sub>. If Q = K<sub>sp</sub>, the rate of dissolution is equal to the rate of precipitation; the solution is saturated, and no net change in the amount of dissolved solid will occur. </br>

The process of calculating the value of the ion product and comparing it with the magnitude of the solubility product is a straightforward way to determine whether a solution is unsaturated, saturated, or supersaturated. More important, the ion product tells chemists whether a precipitate will form when solutions of two soluble salts are mixed.


**Example:** The barium sulfate is used in medical imaging of the gastrointestinal tract. Its solubility product is 1.08 × 10<sup>−10</sup> at 25°C, so it is ideally suited for this purpose because of its low solubility when a “barium milkshake” is consumed by a patient. The pathway of the sparingly soluble salt can be easily monitored by X-rays. Will barium sulfate precipitate if 10.0 mL of 0.0020 M Na<sub>2</sub>SO<sub>4</sub> is added to 100 mL of 3.2 × 10<sup>−4</sup> M BaCl<sub>2</sub>? Recall that NaCl is highly soluble in water.


In [6]:
# BaSO4 precipitation check (Q vs Ksp)

Ksp = 1.08e-10

# Given:
V_BaCl2_mL = 100.0
C_BaCl2_M = 3.2e-4          # Ba2+ concentration in BaCl2 solution

V_Na2SO4_mL = 10.0
C_Na2SO4_M = 0.0020         # SO4^2- concentration in Na2SO4 solution

# Total volume after mixing
V_total_L = (V_BaCl2_mL + V_Na2SO4_mL) / 1000.0

# Moles before mixing
moles_Ba2 = (V_BaCl2_mL / 1000.0) * C_BaCl2_M
moles_SO4 = (V_Na2SO4_mL / 1000.0) * C_Na2SO4_M

# Concentrations after mixing (before precipitation)
Ba2_conc = moles_Ba2 / V_total_L
SO4_conc = moles_SO4 / V_total_L

# Ion product
Q = Ba2_conc * SO4_conc

print(f"moles Ba2+ = {moles_Ba2:.2e} mol")
print(f"[Ba2+] = {Ba2_conc:.2e} M")
print(f"moles SO4^2- = {moles_SO4:.2e} mol")
print(f"[SO4^2-] = {SO4_conc:.2e} M")
print(f"Q = [Ba2+][SO4^2-] = {Q:.2e}")
print(f"Ksp = {Ksp:.2e}")

if Q > Ksp:
    print("Because Q > Ksp, BaSO4 will precipitate.")
elif Q < Ksp:
    print("Because Q < Ksp, BaSO4 will NOT precipitate.")
else:
    print("Q = Ksp, the solution is saturated (at equilibrium).")


moles Ba2+ = 3.20e-05 mol
[Ba2+] = 2.91e-04 M
moles SO4^2- = 2.00e-05 mol
[SO4^2-] = 1.82e-04 M
Q = [Ba2+][SO4^2-] = 5.29e-08
Ksp = 1.08e-10
Because Q > Ksp, BaSO4 will precipitate.


**Solution:** The only slightly soluble salt that can be formed when these two solutions are mixed is BaSO<sub>4</sub> because NaCl is highly soluble. The equation for the precipitation of BaSO<sub>4</sub> is as follows:

BaSO<sub>4</sub>(s) ⇌ Ba<sup>2+</sup>(aq) + SO<sub>4</sub><sup>2−</sup>(aq)

The solubility product expression is as follows:

K<sub>sp</sub> = [Ba<sup>2+</sup>][SO<sub>4</sub><sup>2−</sup>] = 1.08×10<sup>−10</sup>

To solve this problem, we must first calculate the ion product

Q = [Ba<sup>2+</sup>][SO<sub>4</sub><sup>2−</sup>]

using the concentrations of the ions that are present after the solutions are mixed and before any reaction occurs. The concentration of Ba<sup>2+</sup> when the solutions are mixed is the total number of moles of Ba<sup>2+</sup> in the original 100 mL of BaCl<sub>2</sub> solution divided by the final volume (100 mL + 10.0 mL = 110 mL):

moles Ba<sup>2+</sup> = 100 mL (1 L/1000 mL) (3.2×10<sup>−4</sup> mol/1 L) = 3.2×10<sup>−5</sup> mol Ba<sup>2+</sup>

[Ba<sup>2+</sup>] = (3.2×10<sup>−5</sup> mol Ba<sup>2+</sup> / 110 mL) (1000 mL/1 L) = 2.9×10<sup>−4</sup> M Ba<sup>2+</sup>

Similarly, the concentration of SO<sub>4</sub><sup>2−</sup> after mixing is the total number of moles of SO<sub>4</sub><sup>2−</sup> in the original 10.0 mL of Na<sub>2</sub>SO<sub>4</sub> solution divided by the final volume (110 mL):

moles SO<sub>4</sub><sup>2−</sup> = 10.0 mL (1 L/1000 mL) (0.0020 mol/1 L) = 2.0×10<sup>−5</sup> mol SO<sub>4</sub><sup>2−</sup>

[SO<sub>4</sub><sup>2−</sup>] = (2.0×10<sup>−5</sup> mol SO<sub>4</sub><sup>2−</sup> / 110 mL) (1000 mL/1 L) = 1.8×10<sup>−4</sup> M SO<sub>4</sub><sup>2−</sup>

We can now calculate Q:

Q = [Ba<sup>2+</sup>][SO<sub>4</sub><sup>2−</sup>] = (2.9×10<sup>−4</sup>)(1.8×10<sup>−4</sup>) = 5.2×10<sup>−8</sup>

We will now compare Q with the K<sub>sp</sub>.
If Q &gt; K<sub>sp</sub>, then BaSO<sub>4</sub> will precipitate, but if Q &lt; K<sub>sp</sub>, it will not.
Because Q &gt; K<sub>sp</sub>, we predict that BaSO<sub>4</sub> will precipitate when the two solutions are mixed. In fact, BaSO<sub>4</sub> will continue to precipitate until the system reaches equilibrium, which occurs when [Ba<sup>2+</sup>][SO<sub>4</sub><sup>2−</sup>] = K<sub>sp</sub> = 1.08 × 10<sup>−10</sup>.


**Problem:** The solubility product of calcium fluoride (CaF<sub>2</sub>) is 3.45 × 10<sup>−11</sup>. If 2.0 mL of a 0.10 M solution of NaF is added to 128 mL of a 2.0 × 10<sup>−5</sup> M solution of Ca(NO<sub>3</sub>)<sub>2</sub>, will CaF<sub>2</sub> precipitate?


link to paper: https://acp.copernicus.org/articles/24/3405/2024/